rocky/katalogus/views/katalogus.py

```diff
@@ -1,58 +1,24 @@
 from typing import Any, Dict
 
 from account.mixins import OrganizationView
-from django.contrib import messages
-from django.shortcuts import redirect
 from django.urls import reverse
 from django.utils.translation import gettext_lazy as _
-from django.views.generic import FormView, ListView
-from requests import RequestException
+from django.views.generic import FormView, ListView, TemplateView
 
 from katalogus.client import get_katalogus
 from katalogus.forms import KATalogusFilter
 
 
-class KATalogusView(ListView, OrganizationView, FormView):
-    """View of all plugins in KAT-alogus"""
-
-    template_name = "katalogus.html"
+class KATalogusFilterView(FormView):
```

Comment:
I don't understand why `KATalogusFilterView` is created. It isn't used anywhere else than in KATalogusFilterView, so splitting this code over two classes doesn't really add anything? And if the functionality would be moved away from the view it would be a mixin, not a view.